client/modules/IDE/components/Editor.jsx

```diff
@@ -137,9 +137,6 @@ class Editor extends React.Component {
     });
 
     this._cm.getWrapperElement().style['font-size'] = `${this.props.fontSize}px`;
-    this._cm.setOption('indentWithTabs', this.props.isTabIndent);
```

Comment:
because you removed these three options, now they are not set at all! these values should be
```js
{
  indentWithTabs: false,
  tabSize: 2,
  indentUnit: 2
}
```